packages/config/config/devices/0x0063/45604.json

```
@@ -20,5 +20,24 @@
 	"firmwareVersion": {
 		"min": "0.0",
 		"max": "255.255"
+	},
+	"supportsZWavePlus": true,
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "Basic - Local Load",
+			"maxNodes": 5,
+			"description": "Supports Basic Set and is controlled with the local load."
+		},
+		"3": {
+			"label": "Basic - Double Tap", // required
```

```suggestion
			"label": "Basic - Double Tap",
```
Please remove these comments